src/problem/util/randomDistribution.ts

```diff
@@ -1,10 +0,0 @@
-import { randomInt } from "../util/randomizer";
-
-export const randomdist = (distribution:any[]):number=>{
-  var pos = 0;
-  const randomvalue = randomInt(0,100); 
-  while (distribution[pos][1] < randomvalue){
-    pos = pos + 1
-  }
-  return pos
```

Comment:
This is actually a bug; it returns the pos instead of the weighted value in the distribution array. I decided to replace this utility with the "weighted" library.